docs/src/main/paradox/migration-guide/migration-guide-10.1.x.md

```diff
@@ -0,0 +1,25 @@
+# Migration Guide to and within Akka HTTP 10.1.x
+
+## General Notes
+
+See the general @ref[compatibility guidelines](../compatibility-guidelines.md).
+
+## Akka HTTP 10.0.11 - > 10.1.0
+
+### Depend on akka-stream explicitly
+
+Starting from Akka Http 10.1.0, an explicit dependency to the akka-stream module is necessary. See
+@ref[Compatibility with Akka](../compatibility-guidelines.md#compatibility-with-akka) for more information. The minimum
+Akka version currently required is Akka 2.5.11.
+
+### Removal of deprecated methods
+
+When still compiling with 10.0.11 make sure not to refer to deprecated methods. The deprecation notices usually give hints
+about alternatives.
```

Comment:
Maybe we can add a note that removed methods are the ones prior to 10.0.11.

Reply:
Yep, good point.